src/common/lib/api/index.js

```diff
@@ -21,19 +21,27 @@ const apiClientRequest = axios.create({
   }
 })
 
+
+const responseValidation = (response) => {
+  if (response?.status === 503) {
+    VueRouter.push({ name: "maintenance" })
+  } else if (String(response?.status)[0] == 4 || String(response?.status)[0] == 5) {
+    VueRouter.push({ query: { error: true } })
+  }
```

Comment:
Looks good, but can we make these conditions to one line?

Reply:
it has been tested and works as it should. looks like it can be made, update to one line?

Reply:
yes please

Reply:
updated @bukandicki @agustinustheo